ur_robot_driver/src/ur/ur_driver.cpp

```diff
@@ -266,13 +266,15 @@ bool UrDriver::sendScript(const std::string& program)
                              "should not happen.");
   }
 
+  auto program_with_newline = program + '\n';
+
   size_t len = program.size();
   const uint8_t* data = reinterpret_cast<const uint8_t*>(program.c_str());
```

Comment:
Wouldn't you need to replace `program` here with `program_with_newline` as well?

Reply:
indeed...